storm-core/src/ui/public/templates/topology-page-template.html

```diff
@@ -131,33 +131,37 @@
     </tbody>
   </table>
 </script>
+
+<script id="topology-visualization-container-template" type="text/html">
```

Comment:
For the visualization to initialize, it needs to call the old nimbus thrift API call getTopologyInfo, which has not changed.  I only wanted the visualization to initialize when the user shows it, and not pull down all the stream data for every executor each time the page loads.  If we did that, we would lose most of the gains from aggregating stats data on nimbus.